### PR TITLE
V2 fix screen reader new messages

### PR DESCRIPTION
**Pull request by elopezanaya**
Implementation of ability for screen readers to narrate new messages when passing from minimize state to max.

`live chat button  button
Omnichannel Live Chat Widget  document  
Bot WC said:elo@gmail.comOpens in a new window; external. Sent at August 26 at 3:58 PM  Bot WC said:123-43-5678 Sent at August 26 at 3:58 PM  
Snip & Sketch
Snip & Sketch  window
Annotation Canvas  grouping
pane`

corroborated with NVDA, JAWS and Narrator

### CODE REVIEW

File: chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx [hunk 1]
```diff
@@ -292,6 +293,12 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
 
     // Reset the UnreadMessageCount when minimized is toggled and broadcast it.
     useEffect(() => {
+        if (state.appStates.isMinimized){
```
Comment (sarojkpr) [inline chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx:296]: nit:spacing

File: chat-widget/src/components/livechatwidget/common/startChat.ts [hunk 2]
```diff
@@ -166,8 +168,10 @@ const initStartChat = async (chatSDK: any, chatConfig: ChatConfig | undefined, g
         await updateSessionDataForTelemetry(chatSDK, dispatch);
 
         // Set app state to Active
-        if (isStartChatSuccessful) {
+        if (isStartChatSuccessful) {         
+            ActivityStreamHandler.uncork();
             dispatch({ type: LiveChatWidgetActionType.SET_CONVERSATION_STATE, payload: ConversationState.Active });
+
```
Comment (sarojkpr) [inline chat-widget/src/components/livechatwidget/common/startChat.ts:174]: nit:newline

File: chat-widget/src/components/livechatwidget/common/ActivitySubscriber/PauseActivitySubscriber.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    public async apply(activity: any): Promise<void> {
+        await ActivityStreamHandler.restorePromise;
+        return activity;
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any
+    public applicable(activity: any): boolean {
+
```
Comment (sarojkpr) [inline chat-widget/src/components/livechatwidget/common/ActivitySubscriber/PauseActivitySubscriber.ts:17]: nit:newline

**Review by sarojkpr — APPROVED** [review]

File: chat-widget/src/components/livechatwidget/common/Deferred.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+export class Deferred<T> {
+
+    private _promise: Promise<T>;
+   
+    private _resolve!: (value: T | PromiseLike<T>)=> void;
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    private _reject: (reason?: any) => void =()=>{return;};
```
Comment (charliewang95) [inline chat-widget/src/components/livechatwidget/common/Deferred.ts:8]: Please double check all spacing issues like in this line.

File: chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx [hunk 5]
```diff
@@ -292,6 +293,13 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
 
     // Reset the UnreadMessageCount when minimized is toggled and broadcast it.
     useEffect(() => {
+
```
Comment (charliewang95) [inline chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx:296]: Nit: Remove this line

**Review by charliewang95 — APPROVED** [review]

**Pull request merged**